README.md

```diff
@@ -5,495 +5,21 @@
 
 ### If you are looking for the v18.2 branch, please follow this link: [https://github.com/DevExpress/devextreme-vue/tree/18.2](https://github.com/DevExpress/devextreme-vue/tree/18.2).
```

Comment:
We don not need this note in case the Readme does not contain any documentation.